### ISSUE TITLE
rancher webui stop working after upgrade 

### ISSUE THREAD

**Issue by fedya** [rancher/rancher#3948]
Rancher Version: v0.63.0
Docker Version: 1.10.2
OS:  OpenMandriva 2014.0

Steps to Reproduce: run 0.59.0 rancher, upgrade to 0.63.0
Results: webui not working
Expected: webui working

```
[~] # docker run -ti --volumes-from rancher-data --restart=always -p 8080:8080 rancher/server:latest 
Unable to find image 'rancher/server:latest' locally
latest: Pulling from rancher/server
8387d9ff0016: Already exists 
3b52deaaf0ed: Already exists 
4bd501fad6de: Already exists 
a3ed95caeb02: Already exists 
1dd2ffa08394: Already exists 
e4d3f7dc0a74: Already exists 
ba7fdff4f4ab: Already exists 
60029cde7fd0: Already exists 
6c7b3cb5595a: Already exists 
f8a069910cd0: Already exists 
b475b67032eb: Already exists 
752a9eec150c: Already exists 
4504788775b7: Already exists 
Digest: sha256:560b361e5fdb6c419d57c843b6065365d14bb9a0ddc7e333c7abf057a16f1ba8
Status: Downloaded newer image for rancher/server:latest
160312 23:20:15 [Note] /usr/sbin/mysqld (mysqld 5.5.47-0ubuntu0.14.04.1-log) starting as process 35 ...
Uptime: 2  Threads: 1  Questions: 2  Slow queries: 0  Opens: 34  Flush tables: 1  Open tables: 26  Queries per second avg: 1.000
Setting up database
CATTLE_AGENT_PACKAGE_AGENT_BINARIES_URL=/usr/share/cattle/artifacts/agent-binaries.tar.gz
CATTLE_AGENT_PACKAGE_CADVISOR_URL=/usr/share/cattle/artifacts/cadvisor.tar.gz
CATTLE_AGENT_PACKAGE_HOST_API_URL=/usr/share/cattle/artifacts/host-api.tar.gz
CATTLE_AGENT_PACKAGE_NODE_AGENT_URL=/usr/share/cattle/artifacts/node-agent.tar.gz
CATTLE_AGENT_PACKAGE_PYTHON_AGENT_URL=/usr/share/cattle/artifacts/python-agent.tar.gz
CATTLE_AGENT_PACKAGE_RANCHER_DNS_URL=/usr/share/cattle/artifacts/rancher-dns.tar.gz
CATTLE_AGENT_PACKAGE_RANCHER_METADATA_URL=/usr/share/cattle/artifacts/rancher-metadata.tar.gz
CATTLE_AGENT_PACKAGE_RANCHER_NET_URL=/usr/share/cattle/artifacts/rancher-net.tar.gz
CATTLE_CATTLE_VERSION=v0.155.0
CATTLE_DB_CATTLE_DATABASE=mysql
CATTLE_DB_CATTLE_MYSQL_HOST=localhost
CATTLE_DB_CATTLE_MYSQL_NAME=cattle
CATTLE_DB_CATTLE_MYSQL_PORT=3306
CATTLE_DB_CATTLE_USERNAME=cattle
CATTLE_GRAPHITE_HOST=
CATTLE_GRAPHITE_PORT=
CATTLE_HOME=/var/lib/cattle
CATTLE_HOST_API_PROXY_MODE=embedded
CATTLE_LOGBACK_OUTPUT_GELF_HOST=
CATTLE_LOGBACK_OUTPUT_GELF_PORT=
CATTLE_RANCHER_COMPOSE_VERSION=v0.7.2
CATTLE_RANCHER_SERVER_IMAGE=v0.63.0
CATTLE_USE_LOCAL_ARTIFACTS=true
DEFAULT_CATTLE_API_UI_INDEX=//releases.rancher.com/ui/0.94.0
DEFAULT_CATTLE_CATALOG_EXECUTE=true
DEFAULT_CATTLE_COMPOSE_EXECUTOR_EXECUTE=true
DEFAULT_CATTLE_MACHINE_EXECUTE=true
DEFAULT_CATTLE_RANCHER_COMPOSE_DARWIN_URL=https://releases.rancher.com/compose/beta/v0.7.2/rancher-compose-darwin-amd64-v0.7.2.tar.gz
DEFAULT_CATTLE_RANCHER_COMPOSE_LINUX_URL=https://releases.rancher.com/compose/beta/v0.7.2/rancher-compose-linux-amd64-v0.7.2.tar.gz
DEFAULT_CATTLE_RANCHER_COMPOSE_WINDOWS_URL=https://releases.rancher.com/compose/beta/v0.7.2/rancher-compose-windows-386-v0.7.2.zip
23:20:19.277 [main] WARN  o.e.jetty.security.SecurityHandler - ServletContext@o.e.j.w.WebAppContext@56d5cb20{/,file:/usr/share/cattle/cb6bab72d3fdd8af16f1c72bb3366250/,STARTING}{file:/usr/share/cattle/cb6bab72d3fdd8af16f1c72bb3366250/} has uncovered http methods for path: /
23:20:20.282 [main] INFO  ConsoleStatus - [1/6] [0ms] [0ms] Loading config-bootstrap
23:20:20.451 [main] INFO  ConsoleStatus - [2/6] [171ms] [167ms] Loading base-config
23:20:20.744 [main] INFO  ConsoleStatus - [3/6] [464ms] [293ms] Loading config
23:20:20.752 [main] INFO  ConsoleStatus - [4/6] [472ms] [7ms] Starting config-bootstrap
23:20:20.974 [main] INFO  ConsoleStatus - [5/6] [694ms] [221ms] Starting base-config
23:20:20.974 [main] INFO  ConsoleStatus - [6/6] [694ms] [0ms] Starting config
2016-03-12 23:20:21,241 INFO    [main] [ConsoleStatus] [1/32] [0ms] [0ms] Loading bootstrap 
2016-03-12 23:20:21,345 INFO    [main] [ConsoleStatus] [2/32] [104ms] [104ms] Loading config-defaults 
```

On my server 8080 port i see ERR_CONNECTION_REFUSED

**Comment by fedya** [comment]
I think something wrong in data container, because of clean server working fine.

**Comment by fedya** [comment]
```
==> cattle-error.log <==
        at org.mariadb.jdbc.internal.protocol.AbstractQueryProtocol.executeQuery(AbstractQueryProtocol.java:334) ~[mariadb-java-client-1.3.4.jar:na]
        at org.mariadb.jdbc.MariaDbStatement.execute(MariaDbStatement.java:280) ~[mariadb-java-client-1.3.4.jar:na]
        ... 84 common frames omitted
Caused by: java.io.IOException: Last stream not finished
        at org.mariadb.jdbc.internal.stream.PacketOutputStream.startPacket(PacketOutputStream.java:65) ~[mariadb-java-client-1.3.4.jar:na]
        at org.mariadb.jdbc.internal.stream.PacketOutputStream.startPacket(PacketOutputStream.java:78) ~[mariadb-java-client-1.3.4.jar:na]
        at org.mariadb.jdbc.internal.packet.send.SendTextQueryPacket.send(SendTextQueryPacket.java:101) ~[mariadb-java-client-1.3.4.jar:na]
        at org.mariadb.jdbc.internal.protocol.AbstractQueryProtocol.sendQuery(AbstractQueryProtocol.java:383) ~[mariadb-java-client-1.3.4.jar:na]
        ... 87 common frames omitted
```

I found only this
https://github.com/tananaev/traccar/issues/675

**Comment by fedya** [comment]
Well unlocking database not helped me

`2016-03-13 01:11:03,025 ERROR [:] [] [] [] [main           ] [liquibase                           ] classpath:db/changelog.xml: db/core-076.xml::dump1::wizardofmath (generated): Change Set db/core-076.xml::dump1::wizardofmath (generated) failed.  Error: Error executing SQL ALTER TABLE cattle.process_execution ADD created datetime NULL: Can't create table 'cattle.#sql-23_7' (errno: -1) liquibase.exception.DatabaseException: Error executing SQL ALTER TABLE cattle.process_execution ADD created datetime NULL: Can't create table 'cattle.#sql-23_7' (errno: -1)`

**Comment by alena1108** [comment]
@fedya could you check how many records your process_execution table has

**Comment by ibuildthecloud** [comment]
@fedya Sometimes that table is way too big.  It's kind of a catch 22 we created here.  We added a column to that table so we can purge it based on time, but if the table is too big the migration fails to add the column so we can clean the table....

Anyhow, typically a simple fix.  Just `TRUNCATE process_execution`

**Comment by fedya** [comment]
@ibuildthecloud
What i did:
run previous working version of rancher
`# docker run -ti --volumes-from rancher-data -p 8080:8080 rancher/server:v0.59.1`

connected to container with
`docker exec -ti 35fa4cb02d76 mysql`
```
# mysql> TRUNCATE process_execution;
Query OK, 0 rows affected (4.82 sec)
```
And i have this if i run latest release image:

> 2016-03-13 12:47:07,218 ERROR [:] [] [] [] [main           ] [liquibase                           ] classpath:db/changelog.xml: db/core-079.xml::dump1::alena (generated): Change Set db/core-079.xml::dump1::alena (generated) failed.  Error: Error executing SQL ALTER TABLE cattle.environment ADD health_state VARCHAR(128) NULL: Can't create table 'cattle.#sql-23_7' (errno: -1) liquibase.exception.DatabaseException: Error executing SQL ALTER TABLE cattle.environment ADD health_state VARCHAR(128) NULL: Can't create table 'cattle.#sql-23_7' (errno: -1)


if this rancher/server:v0.61.0-rc1
all working as expected

@alena1108 

```
mysql> SELECT COUNT(*) FROM process_execution;
+----------+
| COUNT(*) |
+----------+
|      170 |
+----------+
1 row in set (0.00 sec)

```

**Comment by fedya** [comment]
fixed with 0.63.1
